### PR TITLE
Fix MXE builds

### PR DESCRIPTION
**Pull request by elpaso**

### CODE REVIEW

File: ms-windows/mxe/build-mxe.sh [hunk 1]
```diff
@@ -81,6 +81,7 @@ if [[ "$COMMAND" != *"package"* ]]; then
         -DWITH_SERVER=OFF \
         -DWITH_BINDINGS=FALSE \
         -DQT_LRELEASE_EXECUTABLE=${MXE}/usr/${TARGET}/qt5/bin/lrelease \
+	-DMXE=ON \
```
Comment (m-kuhn) [inline ms-windows/mxe/build-mxe.sh]: Indentation

**Comment by m-kuhn** [comment]
Is this about MXE?

> Methoxetamine, abbreviated as MXE, is a dissociative hallucinogen that has been sold as a designer drug.[1][2] It differs from many dissociatives such as ketamine and phencyclidine (PCP) that were developed as pharmaceutical drugs for use as general anesthetics in that it was designed for grey market distribution.[2][3] 

Source https://en.wikipedia.org/wiki/Methoxetamine

**Comment by elpaso** [comment]
> Is this about MXE?
> 
> > Methoxetamine, abbreviated as MXE, is a dissociative hallucinogen that has been sold as a designer drug.[1][2] It differs from many dissociatives such as ketamine and phencyclidine (PCP) that were developed as pharmaceutical drugs for use as general anesthetics in that it was designed for grey market distribution.[2][3]
> 
> Source https://en.wikipedia.org/wiki/Methoxetamine

You really looks like an expert! That would explain your recent hallucinations about Trump-Riding-Unicorn-Musician, or maybe it is just the code I write that is a powerful hallucinogen ;)

That's my sort of drug btw: https://github.com/qgis/QGIS/tree/master/ms-windows/mxe

**Comment by m-kuhn** [comment]
As long as it's not advertisement it should be fine in my humble opinion.

If someone feels offended or the PSC thinks it's a no go just speak up and it will disappear in no time :cloud:

**Pull request merged**